tests/restv1/PayPal/SingleTicketCest.php

```diff
@@ -62,28 +63,31 @@ public function should_allow_getting_a_ticket_information_by_ticket_post_id( Res
 		$I->seeResponseIsJson();
 
 		$expectedJson = array(
-			'id'                            => $ticket_id,
-			'post_id'                       => $post_id,
-			'global_id'                     => $repository->get_ticket_global_id( $ticket_id ),
-			'global_id_lineage'             => $repository->get_ticket_global_id_lineage( $ticket_id ),
-			'author'                        => $ticket_post->post_author,
-			'status'                        => $ticket_post->post_status,
-			'date'                          => $ticket_post->post_date,
-			'date_utc'                      => $ticket_post->post_date_gmt,
-			'modified'                      => $ticket_post->post_modified,
-			'modified_utc'                  => $ticket_post->post_modified_gmt,
-			'rest_url'                      => $ticket_rest_url,
-			'provider'                      => 'tribe-commerce',
-			'title'                         => $ticket_post->post_title,
-			'description'                   => $ticket_post->post_excerpt,
-			'image'                         => $repository->get_ticket_header_image( $ticket_id ),
-			'available_from'                => $repository->get_ticket_start_date( $ticket_id ),
+			'id'                        => $ticket_id,
+			'post_id'                   => $post_id,
+			'global_id'                 => $repository->get_ticket_global_id( $ticket_id ),
+			'global_id_lineage'         => $repository->get_ticket_global_id_lineage( $ticket_id ),
+			'author'                    => $ticket_post->post_author,
+			'status'                    => $ticket_post->post_status,
+			'date'                      => $ticket_post->post_date,
+			'date_utc'                  => $ticket_post->post_date_gmt,
+			'modified'                  => $ticket_post->post_modified,
+			'modified_utc'              => $ticket_post->post_modified_gmt,
+			'rest_url'                  => $ticket_rest_url,
+			'provider'                  => 'tribe-commerce',
+			'title'                     => $ticket_post->post_title,
+			'description'               => $ticket_post->post_excerpt,
```

Comment:
This should be post_content, as per https://github.com/moderntribe/event-tickets/pull/1140